shells/pipes-shell/source/verbs/process.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {generateId} from '../../../../modalities/dom/components/generate-id.js';
+import {Utils} from '../../../lib/utils.js';
+import {recipeByName, marshalOutput} from '../lib/utils.js';
+import {logsFactory} from '../../../../build/runtime/log-factory.js';
+import {Stores} from '../../../lib/stores.js';
+import {Schemas} from '../schemas.js';
+import {portIndustry} from '../pec-port.js';
+
+const {warn} = logsFactory('pipe');
+
+export const process = async ({type, tag, source, name, modality}, tid, bus, composerFactory, storage, context) => {
```

Comment:
i cannot find where this file is being used.

Reply:
`process.js` should not have been checked in, I'll remove it.